app/views/practical_supports/_edit.html.erb

```diff
@@ -1,37 +1,75 @@
+        <%= f.select :source,
+                      options_for_select(practical_support_source_options(support.source),
+                                        support.source) %>
+        <%= f.number_field :amount,
+                            value: f.object.amount.present? ? number_to_currency(f.object.amount.to_f, unit: '', delimiter: '') : nil,
+                            autocomplete: 'off',
+                            prepend: '$',
+                            step: '0.01' %>
+        <%= f.date_field :support_date, autocomplete: 'off' %>
 
-      <%= f.form_group :confirmed do %>
-        <%= f.check_box :confirmed,
-                        id: "practical_support_confirmed_#{support.id}",
-                        label_class: 'tooltip-header-checkbox',
-                        data: { 'tooltip_text': practical_support_confirmed_help_text } %>
-        <%= f.check_box :fulfilled,
-                        id: "practical_support_fulfilled_#{support.id}",
-                        label_class: 'tooltip-header-checkbox',
-                        data: { 'tooltip_text': practical_support_fulfilled_help_text } %>
+        <% if Config.display_practical_support_attachment_url? %>
+          <%= f.text_field :attachment_url,
+                            autocomplete: 'off',
+                            placeholder: t('activerecord.attributes.practical_support.attachment_url') %>
+        <% end %>
+
+        <%= f.form_group :confirmed do %>
+          <%= f.check_box :confirmed,
+                          id: "practical_support_confirmed_#{support.id}",
+                          label_class: 'tooltip-header-checkbox',
+                          data: { 'tooltip_text': practical_support_confirmed_help_text } %>
+          <%= f.check_box :fulfilled,
+                          id: "practical_support_fulfilled_#{support.id}",
+                          label_class: 'tooltip-header-checkbox',
+                          data: { 'tooltip_text': practical_support_fulfilled_help_text } %>
+        <% end %>
       <% end %>
-    <% end %>
+    </div>
+  </div>
+
+  <div class="tab-pane fade" id="nav-notes" role="tabpanel" aria-labelledby="nav-notes-tab">
+    <div class="row notes-form">
+      <div class="col">
+        <h3 class="mt-5"><%= t('patient.notes.title') %></h3>
+        <%= bootstrap_form_with model: [support, note],
+                                html: { id: 'notes-form-practical-support' },
+                                local: false do |f| %>
+          <%= f.text_area :full_text,
+                          size: '20x5',
+                          placeholder: t('patient.notes.placeholder'),
+                          hide_label: true,
+                          class: 'col' %>
+
+          <div class="row">
+            <div class="col info-form-left">
+              <%= f.submit t('patient.notes.submit'), class: "btn btn-primary" %>
+            </div>
+          </div>
+        <% end %>
+
+        <div class="mt-5">
```

Comment:
I had a hard time getting this to overflow properly. It's not mega janky but it's definitely kinda ugly. I'm fine leaving it but if there's something easy I'm missing that'd be killer.